calendar-helper/index.js

```diff
@@ -46,6 +47,30 @@ class CalendarHelper {
 		this.options.startWeekDay = startDay;
 	}
 
+	setShortcutButtons(shortcutButtons) {
+		this.options.shortcutButtons = shortcutButtons;
+	}
+
+	addShortcutButtons(page, m) {
+		let menuShortcutButtons = [];
+
+		let currentDate = new Date();
+
+		for (let shortcutButton of this.options.shortcutButtons) {
+			let differenceCurrentDate = shortcutButton.differenceCurrentDate;
+
+			let date = new Date(currentDate);
+			date.setDate(date.getDate() + differenceCurrentDate);
+
+			let buttonLabel = shortcutButton.label,
+				buttonCallbackData = "calendar-telegram-date-" + CalendarHelper.toYyyymmdd(date);
+
+			menuShortcutButtons.push(m.callbackButton(buttonLabel, buttonCallbackData));
```

Comment:
You can get the action as an input parameter, so that you can add any kind of button.

```suggestion
			let buttonLabel = shortcutButton.label;
			let buttonAction = shortcutButton.action;

			menuShortcutButtons.push(m.callbackButton(buttonLabel, buttonAction));
```